platforms/documentation/docs/src/docs/userguide/authoring-builds/build_environment.adoc

```diff
@@ -197,41 +224,13 @@ Additional Java system properties are listed link:https://docs.oracle.com/javase
 In a multi-project build, `systemProp` properties set in any project except the root will be *ignored*.
 Only the root project's `gradle.properties` file will be checked for properties that begin with `systemProp`.
 
-=== Using system properties
-
-The following examples demonstrate how to use system properties:
-
-*Example 1:* Setting system properties with a `gradle.properties` file:
-====
-include::sample[dir="snippets/tutorial/systemProperties/groovy",files="gradle.properties[]"]
-====
-
-*Example 2:* Reading system properties at configuration time:
-====
-include::sample[dir="snippets/tutorial/systemProperties/kotlin",files="init.gradle.kts[];settings.gradle.kts[];build.gradle.kts[tags=system-properties]"]
-include::sample[dir="snippets/tutorial/systemProperties/groovy",files="init.gradle[];settings.gradle[];build.gradle[tags=system-properties]"]
-====
-
-*Example 3:* Reading system properties for consumption at execution time:
-====
-include::sample[dir="snippets/tutorial/systemProperties/kotlin",files="build.gradle.kts[tags=system-properties-task-inputs]"]
-include::sample[dir="snippets/tutorial/systemProperties/groovy",files="build.gradle[tags=system-properties-task-inputs]"]
-====
-
-*Example 4:* Setting system properties from the command line `-D gradle.wrapperUser=username`:
-====
-----
-$ gradle -Dsystem=commandLineValue
-----
-====
-
 [[sec:gradle_configuration_properties]]
 == Gradle properties
 
-Gradle provides several options that make it easy to configure the Java process that will be used to execute your build.
-While it's possible to configure these in your local environment via `GRADLE_OPTS` or `JAVA_OPTS`, it is useful to be able to store certain settings like JVM memory configuration and `JAVA_HOME` location in version control so that an entire team can work with a consistent environment.
+Gradle properties configure Gradle itself and usually have the name `org.gradle.\*`.
```

Comment:
> usually

Is there a Gradle property that doesn't start with `org.gradle`? Every property shown below does.